tcs-service/src/main/java/com/transformuk/hee/tis/tcs/service/service/PersonElasticSearchService.java

```diff
@@ -136,10 +136,6 @@ public Page<PersonViewDTO> searchForPage(String searchQuery,
               continue;
             }
             if (StringUtils.equals(columnFilter.getName(), "programmeMembershipStatus")) {
-              shouldBetweenSameColumnFilter.should(new NestedQueryBuilder("programmeMemberships",
-                  new MatchQueryBuilder("programmeMemberships.programmeMembershipStatus",
-                      value.toString()),
-                  ScoreMode.None));
               continue;
```

Comment:
Given most requests end up including a programme membership status, it would be better to add this to the check on [L#121-122](https://github.com/Health-Education-England/TIS-TCS/pull/1020/files#diff-49f81cca9561db7d6db7c513b2f40a4ba94ba3da3ecb2263f7e272f86faba8a6L121-L122).